twitchpopups.htm

```diff
@@ -1,25 +1,6 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-
-    <script type="text/javascript">
-
-        // Change Limmy to your Twitch channel
-        const twitchChannel = 'Limmy';
-
-        // Your alert background. Default is a vibrant green
-        const alertBg = '#00AA00';
-
-        // Hotseat background colour. Default is a fiery, fiery orange.
-        const hotseatBg = '#aa1100';
-
-        // The emoji that surrounds the hotseat messages.
-        const hotseatEmoji = '🔥';
-
-        // Follow the instructions in README.md
-        
-    </script>
-

```

Comment:
These options being at the top of the file makes it easier to use for people who aren't familiar with coding